jpa/org.eclipse.persistence.jpa.jpql/src/main/java/org/eclipse/persistence/jpa/jpql/tools/GenericSemanticValidatorHelper.java

```diff
@@ -388,6 +389,14 @@ public boolean isCollectionMapping(Object mapping) {
         return (mapping != null) && ((IMapping) mapping).isCollection();
     }
 
+    /**
+     * {@inheritDoc}
```

Comment:
consider removing and relying on "default" override javadoc (applies to few more places)

Reply:
There are a lot of places where @Override + `{@inheritDoc}` is being used and I was just keeping everything uniform. I would have considered making changes to all those places since I agree that @Override should be sufficient, but didn't want to widen the changes here. Maybe a large refactoring can be done to clean up the rest.

Reply:
feel free to cleanup remaining `{@inheritDoc}` through the extra PR. Thanks!